stream_alert_cli/terraform/cloudtrail.py

```diff
@@ -36,6 +36,8 @@ def generate_cloudtrail(cluster_name, cluster_dict, config):
 
     cloudtrail_enabled = modules['cloudtrail'].get('enable_logging', True)
     kinesis_enabled = modules['cloudtrail'].get('enable_kinesis', True)
+    send_to_cloudwatch = modules['cloudtrail'].get('send_to_cloudwatch', True)
```

Comment:
Was True your intended default here? If I understand this correctly, this means all existing cloudtrail modules in users' configs will now send to cloudwatch by default unless the user explicitly overrides it. Maybe that's ok, I don't know. Just wanted to check

Reply:
How do they specify just S3 as a destination? Both?

Reply:
Sending to s3 is required and on by default. I will make the cloudwatch stuff disabled by default as well - thanks!